src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Map.cs

```diff
@@ -147,16 +147,9 @@ private void Dispose(bool disposing)
                 }
             }
 
-            ~MapEnumerator()
```

Comment:
I don't understand how can this fix the problem. It only disables the tests coverage,

Reply:
The tests were failing with null object when GC kicked in.  The taking this out it the null object error went away when GC kicked in.